server/www/templates/partials/user-menu.html

```diff
@@ -1,26 +0,0 @@
-<ul class="account-menu">
-<!-- user account menu -->
-    <li>
-        <a ng-href="{{baseUrl + view.name }}">
-            <img class="avatar" src="http://www.gravatar.com/avatar/{{ currentUser.gravatar || '00000000000000000000000000000000' }}?d=retro" alt="{{ currentUser.realname }}"/>
-            <span class="label">{{currentUser.realname || 'Aaliyah Williams'}}</span>
```

Comment:
When bringing across markup, don't add fallbacks to dummy data. Its too easy for these to make it to production and confuse everyone @middle8media @willdoran